packages/ngx-notifications/src/lib/components/notification/notification.component.scss

```
@@ -0,0 +1,46 @@
+@import '../../../../../../node_modules/@a-ui/core/src/styles/quarks';
+@import '../../../../../../node_modules/ngx-toastr/toastr';
```

```suggestion
@import '~@a-ui/core/src/styles/quarks';
@import '~ngx-toastr/toastr';
```
Met als extra opmerking: hoeveel van de Toastr styling heb je eigenlijk nodig? Kan dat ook weggelaten worden?

---

Kan niet weggelaten worden, de context van de toastr nzo zit daar allemaal in

---

Alright, kan je dan de kleine aanpassing van hierboven nog committen? Dan keur ik de PR goed :-)

---

Dat kan niet, de tilde kun je hier niet gebruiken omdat de tilde zal verwijzen naar de node packages van de lib zelf (en niet de globale).